subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -228,6 +249,71 @@ uint8_t ll_setup_iso_path(uint16_t handle, uint8_t path_dir, uint8_t path_id,
 	}
 #endif
 
+#if defined(CONFIG_BT_CTLR_SYNC_ISO)
+	struct lll_sync_iso_stream *stream;
+	struct ll_sync_iso_set *sync_iso;
+	isoal_sink_handle_t sink_handle;
+	struct lll_sync_iso *lll_iso;
+	struct ll_iso_datapath *dp;
+	uint16_t stream_handle;
+	uint32_t sdu_interval;
+	uint16_t iso_interval;
+	uint8_t burst_number;
+	int err;
+
+	if (path_dir != BT_HCI_DATAPATH_DIR_CTLR_TO_HOST) {
+		/* FIXME: workaround to succeed datapath setup for ISO
+		 *        broadcaster until Tx datapath is implemented, in the
+		 *        future.
+		 */
+		return BT_HCI_ERR_SUCCESS;
+	}
+
+	if (handle < BT_CTLR_SYNC_ISO_STREAM_HANDLE_BASE) {
+		return BT_HCI_ERR_CMD_DISALLOWED;
+	}
+	stream_handle = handle - BT_CTLR_SYNC_ISO_STREAM_HANDLE_BASE;
```

Comment:
Perhaps map from handle to stream_handle via macro. We can revisit after resolving conflicts with #40581

Reply:
Yes.